### PR TITLE
adding azure devops extension in extension index

### PR DESCRIPTION
**Pull request by gauravsaralMs**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by gauravsaralMs** [comment]
@williexu : The failure in Travis CI does not look related to this PR, or may be I am missing something...
Can you please take a look

**Comment by gauravsaralMs** [comment]
@williexu : Please take a look at this PR

File: src/index.json [hunk 1]
```diff
@@ -298,6 +298,50 @@
...
+                        {
+                            "requires": [
+                                    "keyring (~=13.2.1)",
+                                    "python-dateutil (==2.7.3)",
+                                    "vsts (==0.1.20)"
+                            ]
+                        }
+                    ],
+                    "summary": "Tools for managing Azure DevOps.",
+                    "version": "1.0.0rc1"
```
Comment (williexu) [inline src/index.json]: Leave this as 0.1.0 as this is still preview.

Comment (gauravsaralMs) [reply src/index.json]: @williexu : made the requested changes.
The PR build is failing and I am not able to understand the reason, can you please help with debugging the failure?
https://travis-ci.org/Azure/azure-cli-extensions/jobs/471761456

Comment (gauravsaralMs) [reply src/index.json]: @williexu : gentle ping

Comment (williexu) [reply src/index.json]: @gauravsaralMs there are problems with your extension that is causing the CI to fail.
I'd recommend:
1. doing a fresh dev install(which will give you access to `azdev`) of the CLI through: [this](https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md)
2. Installing your extension: `az extension add -s https://azuredevopscli.blob.core.windows.net/releases/azure_devops-0.1.0-py2.py3-none-any.whl`
3. Running: `azdev cli-lint --ci --extensions azure-devops` and debugging to fix all errors you see.

Comment (gauravsaralMs) [reply src/index.json]: @williexu Thanks for looking into it
We haven't specified any maxCliCoreVersion as of now so hopefully I don't need to update anything..
Query: Once your PR https://github.com/Azure/azure-cli/pull/8127 is merged then CI issue will be resolved?  (I read your comment but confirming again)

Comment (williexu) [reply src/index.json]: @gauravsaralMs oops sorry I meant **min** cli core version. My bad.

File: src/index.json [hunk 2]
```diff
@@ -298,6 +298,50 @@
                 "sha256Digest": "1faecf9d3b5955829e00b228e174fa6bf1ba327f0d64c2edfc9d096d2926e8dc"
             }
         ],
+        "azure-devops": [
+            {
+                "downloadUrl": "https://azuredevopscli.blob.core.windows.net/releases/azure_devops-0.1.0-py2.py3-none-any.whl",
+                "filename": "azure_devops-0.1.0-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.49",
```
[the remark this answers is not in the document]
Comment (gauravsaralMs) [reply src/index.json]: @williexu : our extension has no dependency on the breaking commit, it is just that the CI for this repo is broken.
That is what https://github.com/Azure/azure-cli-extensions/pull/457 demonstrates, in https://github.com/Azure/azure-cli-extensions/pull/457 PR there is no change (devops extension is not added) and still CI fails

[the remark this answers is not in the document]
Comment (williexu) [reply src/index.json]: I see, sorry I misinterpreted your comment. Yes, the CI will pass after the PR on the CLI side is through.

**Comment by williexu** [comment]
@gauravsaralMs I opened a new PR to test the changes from the CLI side: https://github.com/Azure/azure-cli-extensions/pull/459

**Pull request closed without merging**